client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -152,21 +152,14 @@ const DesignSelector: FunctionComponent< Props > = ( { showPageSelector = false
 					ullamco laboris nisi ut aliquip ex ea commodo consequat.
 				</div>
 			</animated.div>
-
-			<Portal>
-				<DialogBackdrop
-					visible={ showPageSelector }
-					className="design-selector__page-layout-backdrop"
-				/>
-			</Portal>
-
```

Comment:
Curious if there was a reason why we had this in the first place -- basically to make sure we're not missing anything that would introduce any regressions here? @sirreal @razvanpapadopol

Reply:
This seemed necessary for some of the modal-like behavior. I believe hide on click outside didn't work properly without it.

Reply:
Yes, this is for hide on click outside. But there is no more hide on click outside, as we now allow them to click outside to buttons like the back button and the domain picker button.